packages/machine/src/ethereum/utils/multisend-encoder.ts

```diff
@@ -0,0 +1,12 @@
+import { defaultAbiCoder } from "ethers/utils";
+
+import { MultisigTransaction } from "./types";
+
+const ENCODING = ["uint256", "address", "uint256", "bytes"];
+
+export const encodeTransactions = (txs: MultisigTransaction[]) =>
```

Comment:
I don't like arrow function abuse. IMO this should be:

`export function encodeTransactions(txs: MultisigTransaction[]) {`

Reply:
why is the use of arrow functions "abuse"?